src/openms/include/OpenMS/ANALYSIS/MAPMATCHING/MapAlignmentAlgorithmTreeGuided.h

```diff
@@ -0,0 +1,177 @@
+//    documentation and/or other materials provided with the distribution.
+//  * Neither the name of any author or any participating institution
+//    may be used to endorse or promote products derived from this software
+//    without specific prior written permission.
+// For a full list of authors, refer to the file AUTHORS.
+// --------------------------------------------------------------------------
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL ANY OF THE AUTHORS OR THE CONTRIBUTING
+// INSTITUTIONS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+// OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+// WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+// OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+// ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// --------------------------------------------------------------------------
+// $Maintainer:  Julia Thueringer$
+// $Authors: Julia Thueringer $
+// --------------------------------------------------------------------------
+
+// instead of guard
+#pragma once
+
+// included dependencies
+#include <OpenMS/CONCEPT/ProgressLogger.h>
+#include <OpenMS/DATASTRUCTURES/DefaultParamHandler.h>
+#include <OpenMS/DATASTRUCTURES/BinaryTreeNode.h>
+#include <OpenMS/APPLICATIONS/MapAlignerBase.h>
+#include "MapAlignmentAlgorithmIdentification.h"
+
+namespace OpenMS
+{
+  /**
+    @brief A map alignment algorithm based on peptide identifications from MS2 spectra.
+
+    ID groups with the same sequence in different maps represent points of correspondence between the maps. They are used to evaluate the distances between the maps for hierarchical clustering and form the basis for the alignment.
+    Only the best PSM per spectrum is considered as the correct identification.
+
+    For each pair of maps, the distance is determined based on the intersection of the contained identifications and hierarchically clustered using average linkage.
+    Following the tree, the maps are aligned, resulting in a transformed feature map that contains both the original and the transformed retention times.
+    As long as there are at least two clusters, the alignment is done as follows:\n
+    Of every pair of clusters, the one with the larger 10/90 percentile retention time range is selected as reference for the align() method of @ref OpenMS::MapAlignmentAlgorithmIdentification.
+    align() maps the median retention time of each ID group in the second cluster to the reference retention time of this group.
```

Comment:
replace `maps` with another verb. Could be mistaken as a noun and then the reader becomes confused